images/php/cli/Dockerfile

```diff
@@ -13,15 +13,16 @@ RUN apk update \
         tini \
     && apk add nodejs-npm yarn --update-cache  --repository http://dl-3.alpinelinux.org/alpine/edge/main/ --repository http://dl-3.alpinelinux.org/alpine/edge/community/ \
     && rm -rf /var/cache/apk/* \
-    && curl -L -o /usr/local/bin/composer https://getcomposer.org/download/1.5.2/composer.phar \
+    && curl -L -o /usr/local/bin/composer https://github.com/composer/composer/releases/download/1.5.2/composer.phar \
```

Comment:
Also updated composer phar source to use github as it's more stable than `getcomposer.org` & we're already depending on github for drush & console.